test/nools/mocks.js

```diff
@@ -27,7 +27,7 @@ function aTask(type) {
     appliesTo: type,
     name: `task-${idCounter}`,
     title: [ { locale:'en', content:`Task ${idCounter}` } ],
-    actions: [ { form:'example-form' } ],
+    actions: [ {type: 'report', form:'example-form' } ],
```

Comment:
The `type` isn't mandatory, so I think it would be good to keep a mock with out type and another mock with type defined to cover both scenarios.